src/components/Header.tsx

```diff
@@ -58,9 +58,9 @@ const Header: React.FC = memo(() => {
         </div>
       </form>
       <div>
-        {list.map((elem) => {
-          return <Todo key={elem.id} list={elem} />;
-        })}
+        {list.map((elem) => (
```

Comment:
Provide type to `elem` and rename `elem` to `element`

Reply:
it is of type ListItem